python/tvm/auto_scheduler/search_task.py

```diff
@@ -245,15 +254,18 @@ def tune(self, tuning_options, search_policy=None):
 
         _ffi_api.AutoSchedule(search_policy, tuning_options)
 
-    def apply_best(self, log_file, layout_rewrite_option=None):
+    def apply_best(self, log_file, layout_rewrite_option=LayoutRewriteOption.NO_REWRITE):
```

Comment:
Out of scope: Is it possible to maintain the layout rewrite option in the record? This semantic indicates that you need to know the right layout rewrite option when you want to apply a tuned log.

Reply:
Emm ... Then we should also add this to maybe MeasureInput? I'm thinking this may not be so necessary, since we'll always try to apply layout rewrite on network, and this API may only be used in the replying of standalone ops. The user of this API should know what this means.